mail_server/models/__init__.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from coaster.sqlalchemy import IdMixin, TimestampMixin, BaseMixin, BaseNameMixin, JsonDict
+from coaster.db import db
+from coaster.utils import LabeledEnum
+from flask_lastuser.sqlalchemy import UserBase2, ProfileBase
+from .user import *
+from baseframe import __
+
+
+class Organization(ProfileBase, db.Model):
+    __tablename__ = 'organization'
+
+
+class Campaign(BaseNameMixin, db.Model):
+    __tablename__ = 'campaign'
+    __uuid_primary_key__ = True
+
+    organization_id = db.Column(None, db.ForeignKey('organization.id'), nullable=True)
+    organization = db.relationship(Organization, backref=db.backref('campaigns', cascade='all, delete-orphan'))
+    contact_email = db.Column(db.Unicode(254), nullable=False, unique=True)
+
+
+class MailAccount(BaseMixin, db.Model):
+    __tablename__ = 'mail_account'
+    __table_args__ = (db.UniqueConstraint('organization_id', 'friendly_id'),)
+
+    organization_id = db.Column(None, db.ForeignKey('organization.id'), nullable=True)
+    organization = db.relationship(Organization, backref=db.backref('accounts', cascade='all, delete-orphan'))
+
+    friendly_id = db.Column(db.Unicode(255), nullable=False)
+    email = db.Column(db.Unicode(254), nullable=False, unique=True)
```

Comment:
Is this for incoming email addresses? These should be auto-generated when a new email arrives.